elements/rh-footer/rh-global-footer.ts

```diff
@@ -0,0 +1,110 @@
+import type { ColorPalette } from '@patternfly/pfe-core';
+
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import { colorContextProvider } from '@patternfly/pfe-core/decorators.js';
+
+import style from './rh-footer.css';
+import { responsiveStyles } from './rh-footer-responsive.css.js';
+
+import './rh-footer-copyright.js';
+
+/**
+ * @element 'rh-global-footer'
+ * @csspart base
+ * @csspart base
+ * @slot    logo
+ * @csspart logo
+ * @slot    logo-image
+ * @csspart logo-image
+ * @slot    primary
+ * @csspart primary
+ * @slot    primary-start
+ * @csspart primary-start
+ * @slot    primary-end
+ * @csspart primary-end
+ * @slot    secondary
+ * @csspart secondary
+ * @slot    secondary-start
+ * @csspart secondary-start
+ * @slot    secondary-end
+ * @csspart secondary-end
+ * @slot    links-primary
+ * @csspart links-primary
+ * @slot    links-secondary
+ * @csspart links-secondary
+ * @slot    tertiary
+ * @csspart tertiary
+ */
+@customElement('rh-global-footer')
+export class RhGlobalFooter extends LitElement {
+  static readonly styles = [style, responsiveStyles];
+
+  @colorContextProvider()
+  @property({ reflect: true, attribute: 'color-palette' }) colorPalette: ColorPalette = 'darker';
+
+  render() {
+    return html`
+      <div class="section global-base" part="section base">
```

Comment:
You might be wondering "why are only the class names prefixed with `global`?".  It's because we are using a shared .css file so we unfortunately have to do that.  But it's all internal to the shadowdom so it's not that bad I don't think. :)